build: update roachtest scripts to use --suite instead of tags

RaduBerinde opened the pull request:
Epic: none
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/111882)
<!-- Reviewable:end -->

RaduBerinde committed:
build: update roachtest scripts to use --suite instead of tags

There is one more use (`tag:release_qualification`) inside the Release
Qualification script in TeamCity.

Note that we will not remove tag support until the new flags are
backported on all relevant branches, and even after that we will
provide some basic backward compatibility (e.g. convert a single tag
to a suite).

Epic: none
Release note: None
